### PR TITLE
[ipcamera| add autoTracking channel for reolink camera

### PR DESCRIPTION
**Pull request by rogrun**
This PR adds a new channel for autotracking for reolink cameras.

### CODE REVIEW

File: bundles/org.openhab.binding.ipcamera/README.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ The channels are kept consistent as much as possible from brand to brand to make
 | `animalAlarm`               | Switch | RW | Toggles when an animal is in view. |
 | `audioAlarm`                | Switch | R  | When the camera detects noise above a threshold this switch will move to ON. |
 | `autoLED`                    | Switch | RW |When ON this sets a cameras IR LED to automatically turn on or off. |
+| `autoTracking`              | Switch | RW |Turn the automatic mode for the tracking ON and OFF. |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.ipcamera/README.md:223]:
```suggestion
| `autoTracking`              | Switch | RW |Turn the automatic mode for tracking ON or OFF. |
```

File: bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -3074,4 +3075,10 @@
 			</options>
 		</state>
 	</channel-type>
+
+	<channel-type id="autoTracking" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Auto Tracking</label>
+		<description>Turn the automatic mode for the tracking ON and OFF.</description>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml:3082]: Don;t forget to run the i18n tool after adjusting this description.
```suggestion
		<description>Turn the automatic mode for tracking ON or OFF.</description>
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Minor textual adjustment, otherwise LGTM

File: bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml (hunk 2 above)
Comment (rogrun) [reply bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml]: Thx for the hint. All requested changes are done.

**Review by lsiepel — APPROVED** [review]
Thanks LGTM

**Pull request merged**